__tests__/endpoints/app.test.js

```diff
@@ -206,7 +205,49 @@ describe("POST /api/articles/:article_id/comments", () => {
       .send(newComment)
       .expect(404)
       .then(({ body }) => {
-        console.log("body :>> ", body);
+        expect(body.msg).toBe("Not found");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("201 and returns the updated article", () => {
```

Comment:
This would actually be a 200, as we're not creating any new content, just editing it 🙂